src/core/api.proto

```diff
@@ -223,6 +223,25 @@ message InferRequestHeader
   //@@     request.
   //@@
   repeated Output output = 3;
+
+  //@@  .. cpp:var:: uint32 priority [DLIS-1083]
```

Comment:
Remove DLIS since not meaningful externally and these comments generate the docs.